service/src/main/java/uk/gov/hmcts/reform/fpl/service/sdo/GatekeepingOrderEventNotificationDecider.java

```diff
@@ -32,8 +33,9 @@ public Optional<GatekeepingOrderEvent> buildEventToPublish(CaseData caseData, St
         if (null != sdo.getOrderDoc()) {
             event.order(sdo.getOrderDoc());
             event.languageTranslationRequirement(sdo.getTranslationRequirements());
-            // if we are in the gatekeeping state send the NoP related notifications
-            event.notificationGroup(isInGatekeeping(previousState) ? SDO_AND_NOP : SDO);
+            // if we are in the gatekeeping or gatekeeeping-listing state send the NoP related notifications
+            event.notificationGroup((isInGatekeeping(previousState) || isInGatekeepingListing(previousState))
```

Comment:
is this notification ever gonna be sent during gatekeeping state?

Reply:
Removing check for gatekeeping state